app/src/test/java/nodomain/freeyourgadget/gadgetbridge/service/TestDeviceService.java

```diff
@@ -29,7 +29,7 @@ protected void invokeService(Intent intent) {
         // calling though to the service natively does not work with robolectric,
         // we have to use the ServiceController to do that
         service.onStartCommand(intent, Service.START_FLAG_REDELIVERY, (int) (Math.random() * 10000));
-//        super.invokeService(intent);
+        super.invokeService(intent);
```

Comment:
I wanted to check transliterate support at whole. It is unlikely that someone will be able to break the method - it is written too simple;) But, give me a hour, i check and it...